docs/user-guide/testing/integration_tests.md

```diff
@@ -1,3 +1,11 @@
 # Integration Tests
```

Comment:
I would remove this and rename the file and title to Load Tests, since we don't have any content here. I created #1336 to track this